src/modules/file/file.repository.ts

```diff
@@ -94,6 +94,7 @@ export interface FileRepository {
   findOne(
     fileId: FileAttributes['id'],
     userId: FileAttributes['userId'],
+    deleted: FileAttributes['deleted'],
```

Comment:
having a boolean in the parameters like this is usually not so readable when we use the function if you want to pass a filter make it an object so that when  call findOne it can be:

`const file = await this.filesRepository.findOne(fileId, user.id, {deleted: false});`

instead of:
`const file = await this.filesRepository.findOne(fileId, user.id, false);`

If someone is reading the latter, they have no idea what false is for and they have to check the function/types